src/vm/core_test.zig

```diff
@@ -1422,9 +1424,9 @@ test "CairoVM: computeSegmentsEffectiveSizes should return the computed effectiv
     var vm = try CairoVM.init(std.testing.allocator, .{});
     defer vm.deinit();
 
-    try vm.segments.memory.data.put(Relocatable.new(0, 0), .{ .felt = Felt252.fromInteger(1) });
-    try vm.segments.memory.data.put(Relocatable.new(0, 1), .{ .felt = Felt252.fromInteger(1) });
-    try vm.segments.memory.data.put(Relocatable.new(0, 2), .{ .felt = Felt252.fromInteger(1) });
+    try vm.segments.memory.data.put(Relocatable.new(0, 0), MemoryCell.new(.{ .felt = Felt252.fromInteger(1) }));
```

Comment:
To create `MaybeRelocatable` instances in a less verbose way for `Felt252`, you can use the dedicated `fromU256` function that you can find in the `relocatable.zig` file, just to simplify your life.

Reply:
Will open an issue to refactor that everywhere for all our unit tests :)